### PR TITLE
Added spec sheet properties for maltsters

### PR DESCRIPTION
**Pull request by RossMerr**
#132 Created a ViscosityType for the Viscosity property otherwise just followed @pricelessbrewing  suggestions for the properties types

### CODE REVIEW

**Comment by RossMerr** [comment]
Looking deeper over the FermentableBase, FermentableType and FermentableAdditionType, I'm thinking most of the properties from FermentableType should be also pushed down to the FermentableBase, as they are common to both FermentableType and FermentableAdditionType.

As @krutilin comment should still be relevant.

> FermentableAdditionType is about what we plan to add to the mash. FermentableType is total description for storage and inventory. And Base is common part for FermentableType and FermentableAdditionType.

https://github.com/beerjson/beerjson/pull/111#issuecomment-431335896

So I would move moisture, alpha_amylase, diastatic_power, protein, kolbach_index, max_in_batch and recommend_mash

**Comment by pricelessbrewing** [comment]
Base should be the minimum properties used to identify the product (name, supplier, yield, lovibond). 

Fermentable type is used to fully describe the ingredient.  All of the new properties added should be in fermentable type, not base. 

addition type is what's needed to describe the addition (amount and timing object). 


Kolbach is duplicated in base and type, it should just be in type.

**Comment by RossMerr** [comment]
Ok cool thanks I'll take a look over the Kolbach being duplicated and that clarifies it to me for this PR.

Maybe for another PR but then shouldn't product_id be a required field? It's the only link between FermentableAdditionType and FermentableType. As every time someone is sharing a recipe you'll have one or more FermentableAdditionType and you'll probably want the data in the FermentableType, which goes the same for all the other additions.

File: json/fermentable.json [hunk 1]
```diff
@@ -100,7 +100,50 @@
...
+            },
+            "viscosity": {
+              "description" : "The measure of wort viscosity, typically associated with the breakdown of beta-glucans. The higher the viscosity, the greater the need for a glucan rest and the less suitable for a fly sparge.",
+              "$ref": "measureable_units.json#/definitions/ViscosityType"
+            },
+            "dms_p": {
+              "description" : "The amount of DMS precursors, namely S-methyl methionine (SMM) and dimethyl sulfoxide (DMSO) in the malt which convert to dimethyl sulfide (DMS).",
+              "$ref": "measureable_units.json#/definitions/ConcentrationType"
+            },
+            "fan": {
```
Comment (pricelessbrewing) [inline json/fermentable.json:139]: @RossMerr 

Should there be an additional property for fermentability that got lost in a copy-paste?

Comment (RossMerr) [reply json/fermentable.json:139]: Well originally I added 13 properties onto FermentableBase, I moved 11 into FermentableType as it already had kolbach_index and protein so we got left with 11

I guess it was the protein already being their that's causing the confusion.

**Comment by pricelessbrewing** [comment]
Added fermentability to fermentable.json and approving.

**Pull request merged**